requirements.txt

```diff
@@ -1,2 +1,3 @@
-git+git://github.com/broadinstitute/dig-bioindex.git@master#egg=bioindex
+git+https://github.com/broadinstitute/dig-bioindex.git@master#egg=bioindex
```

Comment:
I'm not sure if git stopped supporting the old protocol syntax or what.  But serverless could not find the bioindex dep without this.

Reply:
I would guess that this is just something odd with whatever weird key it was set up to use. I thought it was the https version that is being deprecated in the future, not the git one